pom.xml

```diff
@@ -264,7 +264,7 @@
                     <xmlOutput>true</xmlOutput>
                     <!-- Optional directory to put spotbugs xdoc xml report -->
                     <xmlOutputDirectory>target/site</xmlOutputDirectory>
-		    <includeFilterFile>dev/spotbugs-include.xml</includeFilterFile>
```

Comment:
irrelevant change?

Reply:
The indent of this tag is wrong, this change is trying to fix it.